### ISSUE TITLE
The name 'ctop' is already taken by at least 3 other projects. Consider rename.

### ISSUE THREAD

**Issue by jdrago999** [bcicen/ctop#73]
I just installed some python script with the same name as was thoroughly confused for a few minutes by what I saw when I ran `ctop`.

`dockertop`, `containertop`, `top-but-for-docker`, etc - probably a non-issue but I'm going to post this here for discussion.

**Comment by vielmetti** [comment]
"All the good names are already taken"

**Comment by gmacario** [comment]
Agree, just did `sudo apt install ctop` on my Ubuntu machine to realize I installed another tool 😭

**Comment by pascalandy** [comment]
As I run this in Docker, I don't mind :-p

**Comment by Webysther** [comment]
On my linux i create a alias to dtop like docker top:

```shell
docker run -ti --name ctop --rm -v /var/run/docker.sock:/var/run/docker.sock quay.io/vektorlab/ctop:latest
```

**Comment by demyxco** [comment]
> top-but-for-docker

I vote for this one.

**Comment by nicolus** [comment]
> Agree, just did `sudo apt install ctop` on my Ubuntu machine to realize I installed another tool 😭

Yeah, it's not so much that the name is already taken, but that the other ctop is already included in Ubuntu (and possibly other distributions) repositories. I'm sure we're not the only ones who did `apt-get intall ctop` upon hearing of this tool and were completely confused.

**Comment by danieldaeschle** [comment]
Isn't ctop for all types of containers? Or only for docker?

**Comment by cmer** [comment]
I suggest `ccTop`. You know... like the band. 🤣

**Comment by azlux** [comment]
Currently, I've add ctop to my apt repo with the name docker-ctop.
https://packages.azlux.fr/

The CLI command stay the same, ctop.

**Comment by azlux** [comment]
Any news on this ticket ?
